beta/examples/tensorflow/classification/accuracy_aware_utils.py

```diff
@@ -0,0 +1,96 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import tensorflow as tf
+import tensorflow_addons as tfa
+
+from tensorflow.python.keras.engine import training_utils
+from tensorflow.python.profiler import trace
+from tensorflow.python.eager import context
+from tensorflow.python.keras import callbacks as callbacks_module
+from tensorflow.python.keras.engine import data_adapter
+
+
+def get_training_validation_functions(train_dataset, validation_dataset, callbacks,
```

Comment:
Can we avoid doing this in the sample, like creating a function that returns two function objects? The samples we provide are the things users usually are looking at when they start working with NNCF. 
I think it is natural to define "train_epoch_fn" and "validate_fn" here and import and use them from main.py. If there is an intersection with the existing names. I can help.

Reply:
After refactoring the Keras API usage example, the user no longer has to define and register the training/validation fns, the internal methods of `keras.Model` are utilized instead.